x/leverage/client/cli/query.go

```diff
@@ -224,33 +216,6 @@ func GetCmdQuerySuppliedValue() *cobra.Command {
 	return cmd
 }
 
-// GetCmdQueryReserveAmount creates a Cobra command to query for the
-// reserved amount of a specific token.
-func GetCmdQueryReserveAmount() *cobra.Command {
```

Comment:
I like the idea of condensing queries, but why remove the specific ones? 
Sometimes the user can need only part of that full information '-'

Reply:
> but why remove the specific ones?

The idea is to reduce the API surface so there's less to maintain - that said, it's possible that this measure is a bit aggressive. We could keep the specific ones as well as the improved market summary.

Reply:
It could increase the gas consumption if someone doesn't need all those fields, but I don't know if that is a big problem

Reply:
The part of less to maintain is definitely significant on all sides...